### ISSUE TITLE
Is the trailer tested with the new truck available in Carla?

### ISSUE THREAD

**Issue by josyulavt** [frankeng/CarlaSemiTruckTrailer#8]
Hi, 
I am playing around with the truck and it works well for me so far, I was wondering if the trailer auto hitches to the new truck in carla https://carla.readthedocs.io/en/latest/catalogue_vehicles/#carla-motors-european-hgv-cab-over-engine-type , I did try this in manual control and it failed, what are the changes I need to make for this to work? thank you!

**Comment by frankeng** [comment]
I do not think so, because I made the hitch principle my self. And probably this build-in Carla truck does not have this blueprint.

**Comment by josyulavt** [comment]
> I do not think so, because I made the hitch principle my self. And probably this build-in Carla truck does not have this blueprint.

That makes sense thank you, can you please upload or send the trailer fbx file, I'll try and see if this can be attached to the carla truck. thank you!
